monkey/infection_monkey/network/network_scanner.py

```diff
@@ -119,5 +110,18 @@ def _is_any_ip_in_subnet(ip_addresses, subnet_str):
                 return True
         return False
 
+    def scan_machine(self, victim):
+        """
+        Scans specific machine using instance scanners
+        :param victim: VictimHost machine
+        :return: Victim or None if victim isn't alive
+        """
+        LOG.debug("Scanning target address: %r", victim)
```

Comment:
Do you think it's a good idea to which process is executing this to the log message, perhaps with `mp.current_process().name`?

Reply:
Notice the log format writes process and thread IDs, thanks to a previous long ago commit from me.